README.md

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Looks like we need to update README also to reflect the changes

Reply:
yep